test/unit/test_util.py

```diff
@@ -135,8 +137,34 @@ def test_sysexit_with_custom_code():
     assert 2 == e.value.code
 
 
-def test_get_cookiecutter_template_dir():
-    result = util._get_cookiecutter_template_dir('foo')
+def test_resolve_template_dir_relative():
+    result = util._resolve_template_dir('foo')
+
     parts = pytest.helpers.os_split(result)
 
     assert ('molecule', 'cookiecutter', 'foo') == parts[-3:]
+
+
+def test_resolve_template_dir_absolute():
+    result = util._resolve_template_dir('/foo/bar')
+
+    parts = pytest.helpers.os_split(result)
+
+    assert ('foo', 'bar') == parts[-2:]
+
+
+def test_process_templates():
```

Comment:
We do have a temp_dir fixture you could use, which would give you a temporary dir to do this work.